src/vl-mapactions-modify-action.js

```diff
@@ -17,7 +17,9 @@ export class VlModifyAction extends VlSelectAction {
     this.addInteraction(this.modifyInteraction);
 
     if (options && options.snapping) {
-      this.addInteraction(new VlSnapInteraction(options.snapping.layer || layer));
```

Comment:
Niet helemaal duidelijk waarom deze code vroeger niet voor problemen gezorgd heeft wanneer `options.snapping` true is en er dus layer opgevraagd wordt van een boolean.

Reply:
Ik vermoed omdat er geen afnemer was die het zo gebruikte?

Reply:
En geen test.

Reply:
Het is trouwens fout nu ook want je moet nog de source doorgeven naar snap interactie. Ik pas het aan.

Reply:
Ik zou bij de unit test "kan snapping aanzetten via opties met als standaard snapping layer de modify action layer" in het verleden problemen verwacht hebben dan.

Reply:
options.snapping.layer || layer stond er

dus 'de meegegeven layer' of 'de layer waarop we tekenen, indien er geen meegegeven layer is'

Reply:
true.layer ==> undefined

Reply:
Inderdaad, bij boolean.x had ik gevoelsmatig eerder een exception verwacht, maar geeft inderdaad undefined waardoor het vroeger dus ook geen probleem gaf.